src/components/TargetForm/styles.js

```diff
@@ -12,14 +12,14 @@ const styles = StyleSheet.create({
   },
   mainContainer: {
     position: 'absolute',
-    backgroundColor: BACKGROUND,
+    backgroundColor: WHITE,
```

Comment:
You can add this on the NavigationContainer to make white as the default backgroundColor of all the screens
```
  <NavigationContainer
      theme={{
        colors: { background: WHITE },
      }}
```